### DISCUSSION TITLE
Issues syncing BG from Wear device to Phone

### DISCUSSION THREAD

**Discussion by meatychunks99** [NightscoutFoundation/xDrip#1941] [Q&A]
Hi,
I'm using xDrip 2110041402 on a OnePlus 3T, paired with a Sony Smartwatch 3, connecting to a Dexcom G6.

The set up is used with 'Force Wear Collection Service' enabled, so that my daughter need only take the watch to school.

As of yesterday evening, after a dropped connection to the transmitter, the watch no longer sends BG data back to the phone. I rely on this as my own phone is set up as a follower to give me alerts during the night.

When this issue has intermittently occurred over the past 3 years we've been using the kit, it has been resolved by selecting 'Reset Wear DB' - this sends the stale reading from the phone back to the watch, and upon receiving the next reading from the transmitter the BG reading is refreshed on the phone. This no longer seems to work.

I read through bug 1511 which details a very similar situation (https://github.com/NightscoutFoundation/xDrip/issues/1511). This was marked as resolved, though there is no documented solution. It appears the OP managed to resolve the issue when prompted with the xDrip option 'Enable Sync to Android Wear OS' on the watch. However, the OP did not know how he triggered this prompt and I can find no documentation relating to it.

An additional poster to the thread suggested disabling Smart Lock on the phone, but it is already disabled in my case.

I have already tried a factory reset of the watch several times with no success. I'm not aware of any updates that have been made to the watch or phone since it was last working yesterday morning - it's almost as if something on the phone is blocking the ability for the watch to sync its BG data.

Any help with this issue would be hugely appreciated.

**Comment by meatychunks99** [comment]
I no-one has any experience of this problem, or any suggestions as to the cause or a fix, may I raise this as a bug/issue?

**Reply by Navid200** [reply]
For some reason, there seems to be more experience with watches in the facebook group.
I have invited them to visit these discussions as well before.
I will do that again.

**Reply by meatychunks99** [reply]
Thanks - I did post over on the Facebook group and did receive a single comment. It seems the issue has now resolved itself: when I tested to see whether treatments would be synced back to the watch, I found that on re-enabling 'force wear' for the first time in 4 days, BG values were now syncing back to the phone.
Happy it's now working, but frustrated at the lack of any understanding as to what the issue was. Not much help for anybody else encountering this problem...
